trueos-handbook/introducing.rst

```diff
@@ -816,18 +918,18 @@ screen in :numref:`Figure %s <vbox4>`.
 
    Virtual Hard Drive - New or Existing
 
-This screen is used to create the virtual hard drive, or the amount of
-disk space available to the virtual machine. If this is your first
-virtual machine, keep the default of
-:guilabel:`Create a virtual hard drive now` and click
-:guilabel:`Create` to go to the screen shown in
-:numref:`Figure %s <vbox5>`. If you have created a virtual machine in
-the past and wish to reuse its disk space, select
-:guilabel:`Use an existing virtual hard drive file` from the drop-down
-menu. Create as many virtual machines as desired. However, if the
-computer is getting low on disk space, consider reusing existing virtual
-hard drives to prevent the physical hard drive from being used up by old
-virtual machines.
+This section is used to create the virtual hard drive, or the amount of
+disk space available to the guest OS installed in the virtual machine.
+If this is the first virtual machine, the default offered by the
+:guilabel:`Create a virtual hard drive now` utility should be fine, then
+click :guilabel:`Create` to go to the screen shown in
+:numref:`Figure %s <vbox5>`. If there are existing virtual machines
+already on the host system, reusing an existing virtual disk by
+selecting :guilabel:`Use an existing virtual hard drive file` from the
+drop-down menu. Create as many virtual machines as desired. On thing to
+note, if the host system is getting low on disk space, consider reusing
```

Comment:
Remove "On(e) thing to note, if ..." and just start the sentence with "If the host system ..."